src/read.c

```diff
@@ -4341,6 +4353,7 @@ static avifBool avifDecoderItemIsAlphaAux(const avifDecoderItem * item, uint32_t
 static avifResult avifMetaFindAlphaItem(avifMeta * meta,
                                         const avifDecoderItem * colorItem,
                                         const avifTileInfo * colorInfo,
+                                        avifDiagnostics * diag,
```

Comment:
Nit: It would be good to move the `diag` parameter to be the last. I am sorry I missed this one in my previous review.

Reply:
I saw that after too, done in https://github.com/AOMediaCodec/libavif/pull/1976 (with your comment below.